templates/artist/artist_detail.html

```diff
@@ -24,23 +24,30 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    {% thumbnail artist.photo.img "500" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
-    {% endthumbnail %}
+    <div class="extra-padding">
+        {% thumbnail artist.photo.img "500" as thumb %}
+            <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
+        {% endthumbnail %}
+    </div>
+    <!-- Genre -->
     {% with genres=artist.genres.all %}
     <div class="break">
         <hr>
     </div>
-        <p>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</p>
+    <div class="uppercase">
+        <h4>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</h4>
     {% endwith %}
-    <p>Location: {{ artist.location }}</p>
+    <h4>Location: {{ artist.location }}</h4>
     <div class="break">
         <hr>
     </div>
+    </div>
     {% if campaign %}
         <!-- Goal -->
-        <h2>Goal</h2>
-        <p>${{ campaign.amount|intcomma }} {{ campaign.reason }}</p>
+        <h2>GOAL</h2>
+        <div class="uppercase">
```

Comment:
Or if we're going to define the class elsewhere anyways, you might as well just apply the class to the `h4`s.